app/javascript/application.js

```diff
@@ -1,3 +1,16 @@
+import * as Sentry from "@sentry/browser";
+
+const sentryDsn = document.querySelector("body").dataset.sentryDsn;
+if (sentryDsn) {
```

Comment:
We may not need to but I wonder if we need to check for the Sentry feature flag - was just testing this locally, I had set the `SENTRY_CLIENT_SIDE_DSN` in my local dev settings, but hadn't enabled the feature flag. The alert went straight through to Sentry with the flag not enabled. Maybe that's fine?

Reply:
Fixed.